src/scripts/deploy.js

```diff
@@ -1,56 +1,90 @@
-const assert = require('assert')
-const { series } = require('async')
-const { default: chalk } = require('chalk')
-const { exec } = require('child_process')
-const { confirm } = require('./utils')
+const assert = require("assert");
+const { series } = require("async");
+const { default: chalk } = require("chalk");
+const { exec } = require("child_process");
+const { confirm } = require("./utils");
 
-require('dotenv').config()
+require("dotenv").config();
 
-const NETWORK = process.env.NETWORK
-assert(NETWORK, 'NETWORK env var is required')
+const NETWORK = process.env.NETWORK;
+assert(NETWORK, "NETWORK env var is required");
 
-const SUBGRAPH = process.env.SUBGRAPH
-assert(SUBGRAPH, 'SUBGRAPH env var is required')
+const SUBGRAPH = process.env.SUBGRAPH;
+assert(SUBGRAPH, "SUBGRAPH env var is required");
 
-console.log(`Prepare subgraph config for network ${NETWORK}`)
-console.log(`  Generate subgraph.yaml`)
+const AUTH_TOKEN = process.env.AUTH_TOKEN;
+assert(AUTH_TOKEN, "AUTH_TOKEN env var is required");
 
-series([
-  (callback) => exec(`mustache config/${NETWORK}.json subgraph.yaml.mustache > subgraph.yaml`, null, callback),
+const VERSION = process.env.VERSION;
+assert(VERSION, "VERSION env var is required");
```

Comment:
Not sure what will this version be, but If this is the version of the project, would it make sense to read from package.json or have it automatically:

https://docs.npmjs.com/cli/v6/using-npm/scripts#packagejson-vars

<img width="741" alt="image" src="https://user-images.githubusercontent.com/2352112/235645337-44753db0-ff25-4165-a878-f7e656c85575.png">

Reply:
Good point - will do